components/Feedback/index.tsx

```diff
@@ -1,77 +1,225 @@
+      {submitted ? (
+        <Typography variant='h5'>
+          Thank you, your feedback has been submitted.
+        </Typography>
+      ) : (
+        <Grid container alignItems='center'>
+          {isPositive != null ? (
+            <Grid container flexDirection='column'>
+              {isPositive ? (
+                <>
+                  <Typography variant='h5' mb={2}>
+                    What was most helpful?
+                  </Typography>
+                  <Input
+                    value={feedback}
+                    multiline
+                    rows={4}
+                    sx={{
+                      backgroundColor: 'rgba(249,250,251,.1)',
+                      p: 1
+                    }}
+                    onChange={e => {
+                      setFeedback(e.target.value)
+                    }}
+                  />
+                </>
+              ) : (
+                <>
+                  <Typography variant='h5' mb={2}>
+                    What can we improve?
+                  </Typography>
+                  <Input
+                    value={feedback}
+                    multiline
+                    rows={4}
+                    sx={{
+                      backgroundColor: 'rgba(249,250,251,.1)'
+                    }}
+                    onChange={e => {
+                      setFeedback(e.target.value)
+                    }}
+                  />
+                  <Accordion
+                    sx={{
+                      mt: 2,
+                      backgroundColor: 'transparent',
+                      boxShadow: 'none'
+                    }}
+                  >
+                    <AccordionSummary
+                      expandIcon={<AddIcon />}
+                      aria-controls='panel1a-content'
+                      id='panel1a-header'
+                      sx={{ fontSize: '16px', pl: 0 }}
+                    >
+                      More details
+                    </AccordionSummary>
+                    <AccordionDetails sx={{ px: 0 }}>
+                      <Typography variant='h5' mt={3} mb={1}>
+                        Can you provide the exact steps you took before
+                        receiving the error? For example, the commands you ran.
+                      </Typography>
+                      <Input
+                        fullWidth
+                        multiline
+                        rows={4}
+                        placeholder='1. I ran npm dev...'
+                        onChange={e => {
+                          setSteps(e.target.value)
+                        }}
+                        value={steps}
+                        sx={{
+                          backgroundColor: 'rgba(249,250,251,.1)',
+                          p: 1
+                        }}
+                      />
+                      <Typography variant='h5' mt={3} mb={1}>
+                        If applicable, what version of Safe are you using? If a
```

Comment:
Can we update the copy to `What version of Safe Smart Account, SDK, or API are you using, if applicable? If a package is related to the error, please provide a version.`?